### PR TITLE
DAF-4380: Remove PiP plan limited video logics

### PR DESCRIPTION
**Pull request by mvn-hienvu-hn**
### What was done (Please be a little bit specific)
- [App side implement](https://github.com/dwango-nfc/dwango-app-ch/pull/1805) 
- [Replace BlackCoverView logics with LimitedPlanCoverView logics](https://github.com/dwango-nfc/betterplayer/commit/8638a38fcd94b432db3c8488ce17930c071913c7)
- [Remove limitedPlanVideoReachEnd logics](https://github.com/dwango-nfc/betterplayer/commit/3344b98270bbb1052c75405f9656a35fd2ae49b5)
- [Remove isExtraVideo logics](https://github.com/dwango-nfc/betterplayer/commit/771ce50c7b4ae885692da566ce921bc39ec97803)
- [Remove setPipSeekButtonsHidden logics](https://github.com/dwango-nfc/betterplayer/pull/53/commits/2dd5e73811500e378341727669add74e13cfe7aa)
### JIRA
https://dw-ml-nfc.atlassian.net/browse/DAF-4380

## Screenshot

https://github.com/dwango-nfc/dwango-app-ch/assets/100773699/752e8437-fc15-4120-8939-0258aea8779a

### CODE REVIEW

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM.

@mvn-hienvu-hn Thanks!

File: ios/Classes/BetterPlayer.h [hunk 1]
```diff
@@ -66,9 +66,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)setPipSeekButtonsHidden:(BOOL) isHidden;
 - (void)setIsDisplayPipButtons:(BOOL) isDisplay;
 - (void)setIsPremiumBannerDisplay:(BOOL) isDisplay;
-- (void)setIsPreparingDatasource:(BOOL)isPreparingDatasource;
-- (void)showBlackCoverView;
-- (void)hideBlackCoverView;
+- (void)setIsPipMode:(BOOL) isPipMode;
```
Comment (noboru-i) [inline ios/Classes/BetterPlayer.h]: I think, `setIsPipMode` is only called from this class.
Could we remove this line from this header file?

**Review by noboru-i — COMMENTED** [review]
@mvn-hienvu-hn @mvn-thanhluu-hn 
Thank you for removing and sorry for late reviewing :bow:

I add a comment. Please check it.

And,
https://github.com/dwango-nfc/betterplayer/pull/40/files Could you roll back these changes or not?

File: ios/Classes/BetterPlayer.h (hunk 1 above)
Comment (mvn-hienvu-hn) [reply ios/Classes/BetterPlayer.h]: @noboru-i 
Sure.

**Comment by mvn-hienvu-hn** [comment]
@noboru-i 
Sorry i can only remove the `setPipSeekButtonsHidden` logics of limited plan video ([2dd5e73](https://github.com/dwango-nfc/betterplayer/pull/53/commits/2dd5e73811500e378341727669add74e13cfe7aa)).

The `isLiveStream` flag i still use in the "fix bug" PR of [DAF-4440](https://dw-ml-nfc.atlassian.net/browse/DAF-4440), and this flag is very useful in the native side, we might use it in the future. 

[DAF-4440]: https://dw-ml-nfc.atlassian.net/browse/DAF-4440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Review by noboru-i — APPROVED** [review]
Thank you for checking and partly applying.
I understand.
LGTM 👍

**Comment by noboru-i** [comment]
@mvn-hienvu-hn 
Oh sorry.
Could you create epic branch and merge this PR into that? 🙏

**Pull request merged**